variables.tf

```diff
@@ -0,0 +1,290 @@
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#########################################
+# Variables related to Resource Group
+#########################################
+
+variable "resource_group_tags" {
```

Comment:
We should only have one set of tags named `tags` those should be passed in to all the modules that are wrapped in this ref arch. Since all these resources which are part of this module will have the same lifecycle I dont see a reason why we should have different tags for each module.